src/api/UssUtils.ts

```diff
@@ -201,6 +201,14 @@ export class UssUtils {
         }
     }
 
+    public static checkAbsoluteFilePath(filePath: string) {
+        if (!filePath.startsWith('/')) {
+            throw new ImperativeError({ msg: "Please check the uss file path. The full file path is required."});
+        } else {
+            return filePath;
```

Comment:
Based on the current usage of the function, I don't think we need to return the filePath, but it doesn't hurt (in case we want to use it in the near future)

No changes requested here 😋